packages/connectors/src/Connector.Shared.ts

```diff
@@ -0,0 +1,77 @@
+export interface Dictionary {
+    [Key: string]: string;
+}
+
+export type ChiliPlatform = 'web' | 'server';
+
+export interface ConnectorRuntimeContext {
+    options: Dictionary;
+    fetch(url: string, init: ChiliRequestInit): Promise<ChiliResponse>;
+    logError(msg: string): void;
+    platform: ChiliPlatform;
+    sdkVersion: string;
+}
+
+export type QueryOptions = {
+    sortOrder: string | null;
+    collection: string | null;
+    filter: string[] | null;
+    pageToken: string | null;
+    pageSize: number;
+    sortBy: string | null;
+};
+
+export interface ChiliRequestInit {
+    /**
+     * A BodyInit object or null to set request's body.
+     */
+    body?: string | null;
+    /**
+     * A Headers object, an object literal, or an array of two-item arrays to set request's headers.
+     */
+    headers?: Dictionary;
+    /**
+     * A cryptographic hash of the resource to be fetched by request. Sets request's integrity.
+     */
+    integrity?: string;
+    /**
+     * A string to set request's method.
+     */
+    method?: string;
```

Comment:
could be probably an enum or string literal?

Reply:
Will address this in follow up PR's. This initial PR is just moving whatever we have to a new location. Created https://chilipublishintranet.atlassian.net/browse/EDT-1248 for this